src/components/PermissionsManagement/PermissionsManagement.jsx

```diff
@@ -40,10 +40,11 @@ function PermissionsManagement({ getAllRoles, roles, auth, getUserRole, userProf
     <div key={`${role}+permission`} className="permissions-management">
       <h1 className="permissions-management__title">User Roles</h1>
       <div key={`${role}_header`} className="permissions-management__header">
-        {userProfile?.role === 'Owner' &&
-          <div key={`${role}_name`} className="role-name-container">
+      {(userProfile?.role === 'Owner' ||
+      userProfile?.permissions?.frontPermissions.some(permission => permission === 'postRole')) &&
```

Comment:
Shouldn't have logic based on hard-coded roles. More than just owners can create roles.

```suggestion
      {props.hasPermission('postRole') &&
```